components/IngredientsList.js

```diff
@@ -0,0 +1,66 @@
+import ingredients from "@/assets/ingredients.json";
+//import flavor from "@/assets/flavors.json";
```

Comment:
You can remove this line

Reply:
@NikRoe Thank you for the comments on our code. We will make the changes and get back to you.